### DISCUSSION TITLE
[Suggestion] Add Python to scripting capabilities

### DISCUSSION THREAD

**Discussion by hayescode** [TabularEditor/TabularEditor3#1131] [Ideas]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
I would like to have the option to create and maintain scripts using Python instead of C#.

**Describe alternatives you've considered**
Learning C# and pasting working python code into chatGPT to give me the C# equivalent.

**Additional context**

**Comment by DBojsen** [comment]
Hi @hayescode 

While we do not have any immidiete plans to add python support right now, we are evaluating it for a future feature.
